src/users/users.service.ts

```diff
@@ -72,20 +103,22 @@ export class UsersService {
                         companyId: createuserDto.companyId, 
                         name: createuserDto.companyName, 
                         tags: undefined, recognitions: undefined,
-                        users: [createuserDto]
+                        users: [createuserDto]  
```

Comment:
I don't think we want to be creating a company if the user's specified company doesn't exist. Also, since company admins create users, we probably want to use the company ID from the JWT token rather than use the company ID specified by the `createuserDto`.

Reply:
will add as rework ticket during code clean up sprint